packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsInfo/PayableDetailsInfo.tsx

```diff
@@ -248,9 +293,13 @@ const PayableDetailsInfoBase = ({
                 </TableRow>
                 {showTags && payable.tags && payable.tags.length > 0 && (
                   <TableRow>
-                    <StyledLabelTableCell>
+                    <RequiredFieldLabel
+                      fieldName="tags"
+                      ocrRequiredFields={ocrFields}
```

Comment:
Probably, would be better to replace `ocrRequiredFields` with `required` property.